### PR TITLE
Adding drive details to home page

### PR DESCRIPTION
**Pull request by AndrewSC208**
Adding drive details to home page

### CODE REVIEW

File: services/platform/server/web-client/src/services/web_rpc.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ import { setUserSettings } from './user_slice';
 let BASE_URL = '';
 
 if (process.env.NODE_ENV === 'development') {
-  BASE_URL = 'http://localhost:8000';
+  BASE_URL = 'http://10.0.0.108:8000';
```
Comment (jgkawell) [inline services/platform/server/web-client/src/services/web_rpc.js:10]: the fight continues

Comment (AndrewSC208) [reply services/platform/server/web-client/src/services/web_rpc.js:10]: lol

Comment (AndrewSC208) [reply services/platform/server/web-client/src/services/web_rpc.js:10]: we just need to make this an env

Comment (AndrewSC208) [reply services/platform/server/web-client/src/services/web_rpc.js]: fixed

**Review by jgkawell — APPROVED** [review]

**Pull request merged**